test/kafo/configuration_test.rb

```diff
@@ -71,5 +71,27 @@ module Kafo
         end
       end
     end
+
+    describe '#migrate_configuration' do
+      it 'migrates values from other configuration' do
+        keys = [:log_dir, :log_name, :log_level, :no_prefix, :default_values_dir,
```

Comment:
these 3 tests could be dried up, I'd suggest adding before block to description that will set all keys like here and in each test just run the migration with different options (nothing, skip, with) and check the result